packages/core/content-type-builder/admin/src/components/AttributeIcon/index.js

```diff
@@ -75,7 +76,11 @@ const AttributeIcon = ({ type, customField, ...rest }) => {
     return null;
   }
 
-  return <IconBox as={Compo} {...rest} aria-hidden />;
+  return (
+    <IconBox height={pxToRem(24)} width={pxToRem(32)} shrink={0} {...rest} aria-hidden>
+      <Box as={Compo} />
+    </IconBox>
+  );
 };
```

Comment:
> Another way could be to wrap each icon in a `Box` and assign the dimensions + flex-shrink on that div instead of on the icon directly. This would allow you to keep the styling away from the icon itself.

@gu-stav is this what you meant?

Reply:
Yes, exactly.